packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -1583,19 +1583,16 @@ class FocusTraversalOrder extends InheritedWidget {
 class FocusTraversalGroup extends StatefulWidget {
```

Comment:
Would benefit from adding references to either the class or the `excludeFocus` member documentation.

Reply:
OK, done.